.travis.yml

```diff
@@ -5,8 +5,10 @@ sudo: false
 cache: bundler
 
 script:
+  - bundle install
```

Comment:
This is done automatically in the `install:` target. Because it's a Ruby build it's implicit.